ui/raidboss/timeline.js

```diff
@@ -78,10 +78,8 @@ class Timeline {
       '--Reset--',
       '--sync--',
       'Start',
-      '^ ?21:',
-      '^(\\^\\.\\{14\\})? ?1B:',
-      '^(\\^\\.\\{14\\})? ?21:',
-      '^(\\^\\.\\{14\\})? ?23:',
+      '^(\\(\\?\\<timestamp\\>\\^\\.\\{14\\}\\)) (1B|21|23):',
```

Comment:
Oof.  Thank you.  I would be happy to land this bit on its own, since it's kind of a bugfix vs the feature that the rest of the pull request is.

Reply:
ok i will make seperate PRs than :)